src/PresentationalComponents/SystemsTable/SystemsTable.js

```diff
@@ -44,22 +45,84 @@ const SystemsTable = ({ systemsFetchStatus, fetchSystems, systems, intl, filters
 
     const fetchSystemsFn = useCallback(() => {
         const options = selectedTags.length && ({ tags: selectedTags.join() });
-        fetchSystems({ ...filters, ...options });
-
+        fetchSystems({
+            ...filterFetchBuilder(filters),
+            ...options
+        });
     }, [fetchSystems, filters, selectedTags]);
 
+    const removeFilterParam = (param) => {
+        const filter = { ...filters, offset: 0 };
+        param === 'text' && setSearchText('');
+        delete filter[param];
+        setFilters(filter);
+    };
+
+    const addFilterParam = (param, values) => {
+        values.length > 0 ? setFilters({ ...filters, offset: 0, ...{ [param]: values } }) : removeFilterParam(param);
+    };
+
     const filterConfigItems = [{
         label: intl.formatMessage(messages.name),
         filterValues: {
             key: 'text-filter',
             onChange: (event, value) => setSearchText(value),
             value: searchText
         }
+    }, {
+        label: FC.hits.title,
+        type: FC.hits.type,
+        id: FC.hits.urlParam,
+        value: `checkbox-${FC.hits.urlParam}`,
+        filterValues: {
+            key: `${FC.hits.urlParam}-filter`,
+            onChange: (event, values) => addFilterParam(FC.hits.urlParam, values),
+            value: filters.hits,
+            items: FC.hits.values
+        }
     }];
 
+    const buildFilterChips = () => {
+        const localFilters = { ...filters };
+        delete localFilters.sort;
+        delete localFilters.offset;
+        delete localFilters.limit;
+
+        const prunedFilters = Object.entries(localFilters);
```

Comment:
aside from the deletes in the beginning, this function looks identical to the rec table filter builder, you know what that means 🙃 

abstract it to common! (specifically lines 91 to 106) great news is, since its working presently, breaking it to abstract shouldn't be too painful!

Reply:
Done!